docs/Querying-Brew.md

```diff
@@ -33,7 +33,7 @@ brew info --json=v1 tig | jq .
 To show full JSON information about all installed formulae:
 
 ```sh
-brew info --json=v1 --all | jq "map(select(.installed != []))"
+brew info --json=v1 --eval-all | jq "map(select(.installed != []))"
```

Comment:
We should avoid `--eval-all` in docs, particularly when in this case `--installed` exists and can avoid the `map` and say so immediately below.

Reply:
Should we remove this section then?

Reply:
@p-linnane Yeh 👍🏻

Reply:
https://github.com/Homebrew/brew/pull/16535